sched/signal/sig_timedwait.c

```diff
@@ -406,7 +406,7 @@ int nxsig_timedwait(FAR const sigset_t *set, FAR struct siginfo *info,
            * that we were waiting for?
            */
 
-          if (sigismember(set, rtcb->sigunbinfo.si_signo))
+          if (nxsig_ismember(set, rtcb->sigunbinfo.si_signo))
```

Comment:
Another case that does not account for failure return values.  They will all be treated as TRUE, that the signal is a member of the set.